deploy/stacks/pipeline.py

```diff
@@ -112,10 +116,10 @@ def __init__(
         self.pipeline_bucket.grant_read_write(iam.AccountPrincipal(self.account))
 
         if self.source == 'github':
```

Comment:
Can we rename this to 'codestar'? This will allow users to use GitLab and BitBucket right?

Reply:
Done, after agreeing offline with @asifma